infra/api/app-config/env-config/environment_variables.tf

```diff
@@ -25,6 +25,9 @@ locals {
     API_JWT_AUDIENCE          = "simpler-grants-api-${var.environment}"
 
     TEST_AGENCY_PREFIXES = "GDIT,IVV,IVPDF,0001,FGLT,NGMS,NGMS-Sub1,SECSCAN"
+
+    # grants.gov services/applications URI.
+    GRANTS_GOV_URI = "https://trainingws.grants.gov:443"
```

Comment:
Heads up - all of the backend data in dev/staging is connected to their test environment, not training. If we expect data in our DB to correlate to anything in their SOAP endpoints, would need to use test instead (OR switch the lower envs to pull data from training which would be a bit of work)